python/CMakeLists.txt

```diff
@@ -226,9 +226,11 @@ function(bundle_arrow_implib library_path)
       COPYONLY)
 endfunction(bundle_arrow_implib)
 
+# Always bundle includes
+file(COPY ${ARROW_INCLUDE_DIR}/arrow DESTINATION ${BUILD_OUTPUT_ROOT_DIRECTORY}/include)
```

Comment:
Does this play well with the conda builds?

Reply:
Yes, in fact this is a "fix" so that a conda-installed version of pyarrow can be used in another project's setup.py (since `pyarrow.get_include()` will need to work both in pip and conda)